openmls/src/group/tests/test_proposal_validation.rs

```diff
@@ -1214,7 +1253,7 @@ fn test_valsem106(ciphersuite: Ciphersuite, backend: &impl OpenMlsCryptoProvider
             // Positive case
             bob_group
                 .process_message(backend, original_update_plaintext)
-                .expect("Unexpected error.");
+                .unwrap();
```

Comment:
There are a lot of these unrelated changes here...

Reply:
I think we agreed that a bit of cleanup is fine during pull requests? Do you want me to revert?

Reply:
It's fine around the code that's being touched but these changes are all over the file (this and others). That makes it harder to review. Let's keep it here. But try to keep these type of changes a little more localised to the changes you are making.